.github/workflows/full-ci.yml

```diff
@@ -144,5 +144,120 @@ jobs:
       - name: Run unit tests
         run: mage unitfull
 
-      - name: Run integration tests
+      - name: Integration tests
         run: mage integrationfull
+
+  full-ci-macOS-ee:
+    if: |
+      (github.event_name == 'push') ||
+      (github.event_name == 'pull_request' &&
+        github.event.action == 'labeled' &&
+        github.event.label.name == 'full-ci') ||
+      (github.event_name == 'pull_request' &&
+        github.event.action == 'synchronize' &&
+        contains(github.event.pull_request.labels.*.name, 'full-ci'))
+    runs-on: macos-12
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          submodules: recursive
+
+      - name: Install dependencies
+        run: |
+          brew install --overwrite go mage
+          brew install python3
+          pip3 install pytest tarantool requests psutil pyyaml netifaces
+
+      - name: Build tt
+        env:
+          TT_CLI_BUILD_SSL: 'static'
+        run: mage build
+
+      - name: Integration tests
+        env:
+          TT_EE_USERNAME: '${{ secrets.TT_EE_USERNAME }}'
+          TT_EE_PASSWORD: '${{ secrets.TT_EE_PASSWORD }}'
+        run: mage integrationee
+
+  full-ci-sdk-ee:
```

Comment:
Please add a comment for "sdk-ee", "ce-ee": what do these suffixes mean?

Reply:
Added.